lib/ash/type/struct.ex

```diff
@@ -271,7 +271,7 @@ defmodule Ash.Type.Struct do
             {:ok, value}
 
           is_struct(value) ->
-            :error
+            {:error, "-> constraints: -> instance_of: #{struct}"}
```

Comment:
We don't have any errors in this format so for now this should also just be `is invalid`